kernel/Cargo.toml

```diff
@@ -70,6 +70,7 @@ default-engine = [
   "parquet/object_store",
   "reqwest",
   "tokio",
+  "tokio/sync"
```

Comment:
why was this needed?

Reply:
Bad merge, my branch still had the first attempt at fixing the concurrency bug, which used that package.